config/mtl.toml

```diff
@@ -393,3 +393,23 @@ count = 15
 
 	# mod_cfg [PAR_0 PAR_1 PAR_2 PAR_3 IS_BYTES CPS IBS OBS MOD_FLAGS CPC OBLS]
 	mod_cfg = [0, 0, 0, 0, 14400, 1114000, 16, 16, 0, 0, 0]
+
+        [[module.entry]]
+        name = "GOOGLE RTC AEC"
+        uuid = "B780A0A6-269F-466F-B477-23DFA05AF758"
+        affinity_mask = "0x3"
+        instance_count = "1"
+        domain_types = "1"
+        load_type = "1"
+        module_type = "10"
+        auto_start = "0"
+        sched_caps = [1, 0x00008000]
```

Comment:
are there any docs anywhere I can look what these fields mean? This is all meaningless to me

Reply:
it is converted from aec_module.binmap in module_binmaps for windows

Reply:
I don't think whatever was in there was for our AEC, that would have been intel's or someone elses AEC. We don't deploy on windows

Reply:
yes, it is not used on windows but it will follow the setting since Linux driver decodes module entry according to windows spec. The used items in module entry are **name and uuid**, others are not used by Linux now.
````
        name = "GOOGLE RTC AEC"  // used by Linux driver
        uuid = "B780A0A6-269F-466F-B477-23DFA05AF758" //use by Linux driver
        affinity_mask = "0x3"  // this module can be used on core 0 and core 1
        instance_count = "1" // only one instance can be enabled
        domain_types = "1"  // DP schedule domain
        load_type = "1" //  loadable module
        module_type = "10" // AEC module
        auto_start = "0" 
        sched_caps = [1, 0x00008000] 

        # pin = [dir, type, sample rate, size, container, channel-cfg]
        pin = [0, 0, 0x8, 0x2, 0x2, 0x1,    // support input format of 16 bit, 1 ch, 48000k
                0, 0, 0x8, 0x2, 0x2, 0x4,      // support input format of 16bit  2 ch, 48000k
                1, 0, 0x8, 0x2, 0x2, 0x1]    // support output format of 16bit  1 ch, 48000k

        # mod_cfg [PAR_0 PAR_1 PAR_2 PAR_3 IS_BYTES CPS IBS OBS MOD_FLAGS CPC OBLS]
        mod_cfg = [0, 0, 0, 0, 49152, 417000, 16, 16, 0, 2085, 0,     // unused by Linux driver
                   1, 0, 0, 0, 49152, 29525000, 16, 16, 0, 147625, 0]
````

Reply:
Ipc4 fw will be packed as a binary can work on both windows and Linux platforms  and the packed format was defined first on windows, so we have to define any module in windows layout.

Reply:
Assuming this is for our module, I can't sign off until I get detailed docs explaining the range values of these fields, are they going to be added anywhere?

Reply:
sure, let's add it in my shop list

Reply:
Ack @cujomalainey  We have documentation in the rimage headers (e.g. in https://github.com/thesofproject/rimage/blob/main/src/include/rimage/sof/user/manifest.h ) and we should add any missing pieces here so that this is standalone and sufficient to fill in any needed data to describe the modules. FYI to @mwasko as well, we need to cover this in sof-docs for doing new modules.

Reply:
@kv2019i  please check https://github.com/thesofproject/sof-docs/pull/451. Thanks!